### PR TITLE
[backend] feature(seed): create pending users on fenv and dev

### PR DESCRIPTION
**Pull request by carinelebas**
# Context
Create pending users on feature env and dev environment

## How to test
Check there are pending users on feature env and dev environments

## Related issues

- Related to #1506 

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [ ] I tested all features and edge cases
- [ ] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [ ] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-api/tests/test-utils.ts [hunk 1]
```diff
@@ -75,12 +75,12 @@ export async function createUser({
   first_name = 'John',
   last_name = 'Doe',
   selected_organization_id = 'ba091095-418f-4b4f-b150-6c9295e232c4',
```
Comment (jpkha) [inline apps/portal-api/tests/test-utils.ts]: Actually, I know I did this but it quite confusing by reading the function.
Should we call it Professional_Organization_Id ? something like that instead of selected but in fact we are not selected in case of pending

File: apps/portal-api/tests/test-utils.ts [hunk 2]
```diff
@@ -106,32 +116,35 @@ export async function createUser({
...
+              user_id: userId,
+              organization_id: userId,
+            },
+          ]
+        : []),
       {
         user_id: userId,
         organization_id: userId,
       },
     ]);
```
Comment (jpkha) [inline apps/portal-api/tests/test-utils.ts]: If it pending is does duplicate the relation user_id and organization_id right ?
Also it should be reversed I think ?

If pending -> do not add the Filigran id.
If not -> add Filigran id (selected_organization_id)

**Review by Kroustille — APPROVED** [review]
